src/main.zig

```diff
@@ -3,12 +3,13 @@ const mem = std.mem;
 const Allocator = mem.Allocator;
 const process = std.process;
 const Compilation = @import("Compilation.zig");
-const Driver = @import("Driver.zig");
+const Driver = @import("driver/Driver.zig");
```

Comment:
I think the proper naming scheme would be for `Driver.zig` to remain in `src` and the other files related to it being in a `Driver` folder.